ui/js/controllers/logviewer.js

```diff
@@ -376,6 +401,21 @@ logViewerApp.controller('LogviewerCtrl', [
             });
         }
 
+        function displayErrorLines(errors) {
+            var css = errors
+              .reduce((line, err) => `${line}a[id="${err.line_number + 1}"]+span, `, '')
```

Comment:
@eliperelman `line` contains the selector (e.g, `a[id="1829"]+span, a[id="4490"]+span, a[id="4492"]+span, a[id="4538"]+span`). Want me to rename it to `selector`